rpc/src/rpc.rs

```diff
@@ -71,7 +72,7 @@ use {
         sysvar::stake_history,
         transaction::{
             self, AddressLoader, MessageHash, SanitizedTransaction, TransactionError,
-            VersionedTransaction,
+            VersionedTransaction, MAX_TX_ACCOUNT_LOCKS,
```

Comment:
How much is `transaction_account_lock_limit` expected to vary? It looks like it's only adjustable for test-validators; is that right?
I lean toward keeping the constant here because (a) I think it would get confusing for the JSON-RPC limit to vary; and (b) it's not obvious which bank we should query for `get_transaction_account_lock_limit` in request processing, since the rpc just uses the fee cache.